.github/composite_actions/log_cw_metric_wrapper/action.yaml

```diff
@@ -53,14 +53,6 @@ inputs:
 runs:
   using: "composite"
   steps:
-    - name: Exit if not scheduled
```

Comment:
Are you sure you want to track metrics for every single test trigger?

Reply:
Yes, at least for now.

Reply:
If we do this shouldn't we track the trigger type? Otherwise we have no means to separate schedules from PR tests that are failing because the PR is half done.

Reply:
Oh that's a great point - let me think on it

Reply:
Done - sending event type now.

Reply:
Where are you sending this event type or other means to distinguish events based on whether they were triggered by a PR / scheduled / etc. ?

Reply:
@fjnoyp line 139 of [log_cw_metric.dart](https://github.com/aws-amplify/amplify-flutter/pull/4058/files#diff-b295e4ced78ed86faa0d0198c1be5404053c1f96a6c1f35a7bbf96a42768fd9c)